_includes/footer.html

```diff
@@ -32,10 +32,17 @@
         <span>Top Contributors</span>
         <a href="{{ cloudicalLink }}">Cloudical</a>
         <a href="{{ cybozuLink }}">Cybozu, Inc</a>
+        <a href="{{ koorLink }}">Koor Technologies, Inc.</a>
         <a href="{{ redHatLink }}">Red Hat</a>
         <a href="{{ suseLink }}">SUSE</a>
         <a href="{{ upboundLink }}">Upbound</a>
       </div>
+      <div class="col_sm-12">
```

Comment:
Let's split this commit into a separate PR so it unblocks the other commit for merge. We need to discuss with maintainers (e.g. in the community meeting) if this is the right place to expose this info.

Reply:
Done, I have created #131